addons/color-picker/code-editor.js

```diff
@@ -0,0 +1,75 @@
+import { normalizeHex, getHexRegex } from "../../libraries/normalize-color.js";
+
+export default async ({ addon, console, msg }) => {
+  let elem;
+  while (true) {
+    elem = await addon.tab.waitForElement("button.scratchEyedropper", { markAsSeen: true });
+    if (addon.tab.editorMode !== "editor") continue;
+    addon.tab.redux.initialize();
+    const getColor = () => {
```

Comment:
We can move this function and `setColor` to a higher scope and take the element as input.